src/archetypes/referencebrowserwidget/browser/popup.pt

```diff
@@ -97,6 +99,14 @@
                i18n:domain="plone"
                i18n:attributes="value label_search;"
                />
+  &nbsp;<span style="font-weight: bold; color: grey; border-bottom: 1px dotted; cursor: help;"
+              title="#"
+              id="searchWildcardHelp"
+              tal:condition="python: search_index in wildcardable_indexes"
+              i18n:attributes="title"
+              data-wildcardableIndexes="#"
+              tal:attributes='title python: widget.use_wildcard_search and "wild_card_search_enabled_help" or "wild_card_search_disabled_help";
+                              data-wildcardableIndexes wildcardable_indexes_as_json'>?</span>
```

Comment:
You should use real message instead of msgid here. English users see wild_card_search_enabled_help here.
Or better define the message in python an use widget/help_message here.
def help_message(self):
  return widget.use_wildcard_search and _("wild_card_search_enabled_help", default="blabla enabled...") or _("wild_card_search_disabled_help", default="blabla disabled...")

so i18ndude can extract them.